e2e/tests/update_params_test.go

```diff
@@ -36,30 +37,28 @@ var updateParamsTxJSONTemplate = template.Must(
 `))
 
 func (s *suite) AllModuleParamsAreSetToTheirDefaultValues(moduleName string) {
-	s.Skip("TODO_TEST: complete step definitions for update_params.feature scenarios")
-
-	args := strings.Split(fmt.Sprintf("query %s params", moduleName), " ")
-	args = append(args, "--output", "json")
-	res, err := s.pocketd.RunCommandOnHost("", args...)
+	argsAndFlags := []string{
+		"query",
+		moduleName,
+		"params",
+		fmt.Sprintf("--%s=json", cometcli.OutputFlag),
+	}
+	res, err := s.pocketd.RunCommandOnHost("", argsAndFlags...)
 	require.NoError(s, err)
 
 	s.Log(res.Stdout)
 
 	switch moduleName {
 	case tokenomicstypes.ModuleName:
 		var tokenomicsParamsRes tokenomicstypes.QueryParamsResponse
-		err = s.cdc.UnmarshalJSON([]byte(res.Stdout), &tokenomicsParamsRes)
-		require.NoError(s, err)
-		require.Equal(s, tokenomicstypes.DefaultParams(), tokenomicsParamsRes.GetParams())
+		s.cdc.MustUnmarshalJSON([]byte(res.Stdout), &tokenomicsParamsRes)
 
 	case prooftypes.ModuleName:
 		var proofParamsRes prooftypes.QueryParamsResponse
-		err = s.cdc.UnmarshalJSON([]byte(res.Stdout), &proofParamsRes)
-		require.NoError(s, err)
-		require.Equal(s, prooftypes.DefaultParams(), proofParamsRes.GetParams())
+		s.cdc.MustUnmarshalJSON([]byte(res.Stdout), &proofParamsRes)
```

Comment:
```suggestion
		s.cdc.MustUnmarshalJSON([]byte(res.Stdout), &proofParamsRes)
		require.Equal(s, prooftypes.DefaultParams(), proofParamsRes.GetParams())
```

Reply:
@Olshansk FYI, adding back an assertion that seems to have gotten dropped.

Reply:
Sg